include/deal.II/grid/grid_tools.h

```diff
@@ -606,6 +606,38 @@ namespace GridTools
   /*@{*/
 
   /**
+   * Given a @p cache and a list of @p points create the quadrature rules. This
+   * function returns a tuple containing the following elements:
+   * - The first ( get<0>), call it cells, is a vector of the cells containing at least one of
+   *  the @p points
+   * - The second, call it qpoints, is a vector of vectors. Every vector corresponds to the
```

Comment:
- a vector `qpoints` of vector of points, containing in `qpoints[i]` the reference positions of all points that fall within the cell `cells[i]`